### PR TITLE
Configure listener queues based on the URI settings or config properties

### LINKED ISSUE #1771: Configurable queue settings for bound_to exchange queues
The settings of bound_to_exchange queues are not configurable. It is currently hard coded as explicit, non durable and auto deleted. Can this be made configurable by getting the values from the settings?

If the queue is exclusive, queue mirroring is not happening in rabbitmq cluster environment. So during failover of rabbitmq nodes, the queues are getting deleted and never created again. 

The fix can be to get the settings of bound_exchange queues also from config properties so that these can be set based on the environment.

### PR DESCRIPTION
**Pull request by msdhivya**
This PR contains changes for configuring queues based on the settings provided in the URI. If no settings are provided, then default settings will be used to create the queue.

Closes #1771

### CODE REVIEW

**Comment by apanicker-nflx** [comment]
@ritu-p Could you please help with reviewing this PR? Thanks

File: contribs/src/main/java/com/netflix/conductor/contribs/queue/amqp/AMQPObservableQueue.java [hunk 1]
```diff
@@ -536,7 +536,8 @@ protected void receiveMessages() {
 			if (useExchange) {
 				// Consume messages from an exchange
 				getOrCreateExchange();
-				// Create queue if not present based on the settings
+				// Create queue if not present based on the settings provided in the queue URI or configuration properties
+				// Default settings if not provided in the queue URI or properties: isDurable: true, autoDelete: false, isExclusive: false
```
Comment (ritu-p) [inline contribs/src/main/java/com/netflix/conductor/contribs/queue/amqp/AMQPObservableQueue.java:540]: Can you put the example url here as to how it is set,
Also mention that this config gets applied to both exchange and queue and we will need a separate key if they need to be separated out

Comment (msdhivya) [reply contribs/src/main/java/com/netflix/conductor/contribs/queue/amqp/AMQPObservableQueue.java]: Thank you @ritu-p . I have updated the comment.
Can you please check?

**Comment by msdhivya** [comment]
Thank you @ritu-p 
@apanicker-nflx one of the coverage check is failing in the build. But it is not related to my changes. I have added Unit tests for the changes I have done. Please advise on what should be done.

**Pull request merged**